### DISCUSSION TITLE
Looking for a new owner!

### DISCUSSION THREAD

**Discussion by DarcyManoel** [EndlessSkyCommunity/-Omnis#59] [Announcements]
To start, I'd like to say that over the years it has been a rewarding experience to see this plugin grow and contribute to the community as a tool for players. However, due to waning interest and changing circumstances, I believe it's time for someone else to take the reins and continue its development. Making this matter public is far later than it should have been, I stopped pushing commits around 8 months ago now.

You would already know what this post is about by the title, so I won't say too much. I'd like to see ongoing commitment from whoever takes over this project. I'd also appreciate if the fundamentals of this plugin not change too drastically, to maintain familiarity with the people who enjoy use of it.

If you are interested in taking over this project, please reach out to me directly on Discord (darcymanoel_). I am happy to provide any support or feedback after the handover.

I look forward to seeing where new ownership will take this plugin in the future.

**Comment by TheGiraffe3** [comment]
I would be willing to add ships/outfits that get added in the base game here (and, whether I get accepted or not, I'll probably still keep PRing new factions when they are merged to vanilla), but I'm not sure if I would be up for creating new content types for Omnis.

**Reply by DarcyManoel** [reply]
Would you be willing to get in touch over Discord for organising this?
